lib/db_helper/tasks.rb

```diff
@@ -11,50 +11,52 @@
-    begin
-      sources = YAML::load(File.open(File.join(ENV['RAILS_ROOT'] || '.', File.join('config','database_import.yml'))))
-      default = sources.delete('default') || {:db => {}}
+    root = ENV['RAILS_ROOT'] || '.'
+    path = File.join(root, 'config', 'db_helper_import_from_remote.yml')
+    sources = YAML.load(File.open(path))
+    default = sources.delete('default') || {db: {}}
 
-      puts "Select database?"
+    puts "Select database?"
 
-      sources.values.each.with_index do |source,i|
-        puts "#{i + 1}. #{source['name']}"
-      end
-
-      num_selected = InputReader.get_int(prompt: 'Enter Choice: ', valid_values: (1..sources.count).to_a)
-      selected = sources.values[num_selected - 1]
-
-      importer = SqlImporter.new(selected, default)
-      importer.backup
-
-      post_execution_commands = Array.wrap(selected.has_key?('post_execution_commands')  ? selected['post_execution_commands'] : default['post_execution_commands'])
+    sources.values.each.with_index do |source,i|
+      puts "#{i + 1}. #{source['name']}"
+    end
 
-      if !post_execution_commands.empty?
-        puts "Post execution commands"
-        post_execution_commands.each.with_index { |c,i| puts "#{i + 1}. #{c}" }
-        if InputReader.get_boolean(:prompt => "Run post execution commands (careful what you do, the command strings will be evaluated and executed!) (y/n)?")
-          puts "Running post execution commands"
-          post_execution_commands.each do |c|
-            eval c
-          end
+    num_selected = InputReader.get_int(prompt: 'Enter Choice: ',
+                                       valid_values: (1..sources.count).to_a)
+    selected = sources.values[num_selected - 1]
+
+    importer = DbHelper::SqlImporter.new(selected, default)
+    importer.backup
+
+    key = 'post_execution_commands'
+    post_execution_commands = Array.wrap(selected.fetch(key, default[key]))
+
+    if !post_execution_commands.empty?
+      puts "Post execution commands"
+      post_execution_commands.each.with_index { |c,i| puts "#{i + 1}. #{c}" }
+      prompt = <<-PROMPT
+        Run post execution commands (careful what you do, the command strings
+        will be evaluated and executed!) (y/n)?
+      PROMPT
+      if InputReader.get_boolean(prompt: prompt)
+        puts "Running post execution commands"
+        post_execution_commands.each do |c|
+          eval c
         end
       end
-    
-
-      puts "Done"
-    #rescue Exception => e
-    #  puts "Error: #{e.message}"
     end
+
+    puts "Done"
   end
 
   # TODO : Clean up this mess.
   desc "Import remote database from hot backup into sandbox"
   task :update_sandbox do
-
     db_config, env = if defined?(Rails)
                        [Rails.configuration.database_configuration, Rails.env]
                      else
-                       [YAML::load(File.read(ENV['YML'])), ENV['ENV']]
+                       [YAML.load(File.read(ENV['YML'])), ENV['ENV']]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.